psi4/driver/procrouting/proc.py

```diff
@@ -1611,10 +1615,13 @@ def run_dfocc_gradient(name, **kwargs):
     elif name in ['ccsd']:
         core.set_local_option('DFOCC', 'WFN_TYPE', 'DF-CCSD')
         core.set_local_option('DFOCC', 'CC_LAMBDA', 'TRUE')
+    elif name in ['ccsd(t)']:
+        core.set_local_option('DFOCC', 'WFN_TYPE', 'DF-CCSD(T)')
+        core.set_local_option('DFOCC', 'CC_LAMBDA', 'TRUE')
     else:
         raise ValidationError('Unidentified method %s' % (name))
 
-    if name in ['mp2', 'mp2.5', 'mp3', 'lccd', 'ccd', 'ccsd']:
+    if name in ['mp2', 'mp2.5', 'mp3', 'lccd', 'ccd', 'ccsd', 'ccsd(t)', 'ccsd(at)']:
```

Comment:
I don't think `ccsd(at)` can be reached here at present? Not in a `select_` fn or in the options setting directly above.